utils/movies.php

```diff
@@ -0,0 +1,49 @@
+<?php 
+require "database.php";
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+// Upload movie function
+
+// example: uploadMovie(1, "title", "asdfg", "p" , 1, "asdfg", 1, 3);
+function uploadMovie($userId, $title, $path, $thumbnailPath, $description, $ageRating, $agerestrictions, $genres){
+    $accepted = getTableRecord("SELECT geverifieerd FROM gebruiker WHERE id = ?", "i", array($userId)   );
+    executeQuery("INSERT INTO film (gebruiker_id, titel, pad, thumbnail_pad, geaccepteerd, beschrijving, kijkwijzer_leeftijd) VALUES (?, ?, ?, ?, ?, ?, ?)", "isssisi", array($userId, $title, $path, $thumbnailPath, $accepted, $description, $ageRating));
+    $filmId = getTableRecords("SELECT MAX(id) FROM film");
+    $filmIdInput = getTableRecord("SELECT film.id FROM film WHERE id = ?", "i", array($filmId[0]['MAX(id)']));
+    
+    foreach($genres as $genre){
+        $genreInput = getTableRecord("SELECT genre.id FROM genre WHERE id = ?", "i", array($genre));
+        executeQuery("INSERT INTO genre_film (genre_id, film_id) VALUES (?, ?)", "ii", array($genreInput['id'], $filmIdInput['id']));
+    }
+    foreach($agerestrictions as $agerestriction){
+        $agerestrictionInput = getTableRecord("SELECT kijkwijzer_geschiktheid.id FROM kijkwijzer_geschiktheid WHERE id = ?", "i", array($agerestriction));
+        executeQuery("INSERT INTO film_kijkwijzer_geschiktheid (kijkwijzer_geschiktheid_id, film_id) VALUES (?, ?)", "ii", array($agerestrictionInput['id'], $filmIdInput['id']));
+    }
```

Comment:
Klopt de benaming? Age is een attribuut van een film, hier lijkt het alsof er meerdere ages aan een film worden gekoppeld, maar dit zouden volgens mij geschiktheiden moeten zijn toch?

Reply:
En moet nog veranderd worden naar engels